d2common/d2interface/app.go

```diff
@@ -0,0 +1,24 @@
+package d2interface
+
+// App is the OpenDiablo2 application
+// As the app starts, it creates all of the AppComponents
+// and binds to them.
+type App interface {
+	Run()
+	BindAppComponent(AppComponent) error
+
+	// The AppComponents
+	Input() (InputManager, error)
+	Audio() (AudioProvider, error)
+	Renderer() (Renderer, error)
+	Terminal() (Terminal, error)
+	Asset() (AssetManager, error)
+}
+
+// AppComponent is part of the app
+// components are first bound, then initialized.
```

Comment:
Either add blank line, or period and capital letter, as `go doc github.com/OpenDiablo2/OpenDiablo2/d2common/d2interface.AppComponent` would otherwise render the docs as follows:

*AppComponent is part of the app components are first bound, then initialized.*

Reply:
good catch, will fix.

Reply:
should be fixed :)